packages/ckeditor5-ui/src/dialog/dialogview.ts

```diff
@@ -260,7 +260,7 @@ export default class DialogView extends /* #__PURE__ */ DraggableViewMixin( View
 			},
 			children: [
 				{
-					tag: 'div',
+					tag: 'form',
```

Comment:
`DialogView` is a "generic" component reused by CKEditor 5 in multiple places by multiple plugins. This change would affect all of the dialogs so this is not the right place for such change if we only want to adjust the behavior of a specific plugins dialog.